Zappy_IA_Report/NN/Code/Tutorial4.py

```
@@ -0,0 +1,31 @@
+import tensorflow as td
```

Ruff (0.11.9)

`tensorflow` imported but unused

Remove unused import: `tensorflow`

(F401)